code/game/objects/items/devices/chameleon_counter.dm

```diff
@@ -0,0 +1,78 @@
+/obj/item/chameleon_counterfeiter
+	name = "chameleon counterfeiter"
+	icon = 'icons/obj/device.dmi'
+	icon_state = "cham_counter"
+	flags = CONDUCT
+	slot_flags = SLOT_BELT
+	item_state = "electronic"
+	w_class = WEIGHT_CLASS_SMALL
+	origin_tech = "syndicate=3;magnets=3"
+	var/can_use = TRUE
+	var/saved_name
+	var/saved_desc
+	var/saved_icon
+	var/saved_icon_state
+	var/saved_item_state
+	var/saved_overlays
+	var/saved_underlays
+	var/activate_dummy = FALSE
```

Comment:
Might be good to change this variable name to something past-tense, like `dummy_active`